handbook/company/values.md

```diff
@@ -110,3 +110,20 @@ Our code and PRs are public, so lack of transparency is only obfuscation, not se
 ### Continuously grow
 
 You strive to continuously grow and learn by genuinely soliciting feedback early and often, and humbly reflecting on your past mistakes.
+
+We value growth at Sourcegraph in the form of continuous learning:
+
+- We actively seek out learning opportunities instead of avoiding them.
+- We identify gaps in our knowledge and share them openly and honestly.
+- We adopt a growth mindset.
```

Comment:
The first sentence here lists two things, but then we follow up with more things, which feels a bit weird and is maybe duplicative. It kind of feels like the first summary sentence needs to change.

Other minor points
- align on "You" vs "we"
- `We adopt a growth mindset.` feels like a circular definition, so recommend removing it

```suggestion
You strive to continuously grow and learn.

- You genuinely solicit feedback early and often.
- You humbly reflect on past mistakes.
- You actively seek out learning opportunities instead of avoiding them.
- You identify gaps in knowledge and share them openly and honestly.
```